assets/js/theme/common/carousel/utils/arrowAriaLabling.js

```diff
@@ -0,0 +1,26 @@
+const NUMBER = '[NUMBER]';
+const integerRegExp = /[0-9]+/;
+const lastIntegerRegExp = /(\d+)(?!.*\d)/;
+
+export default ($prevArrow, $nextArrow, realCurrentSlide, realSlideCount) => {
+    if (realSlideCount < 2) return;
```

Comment:
what's your thought on using name actualSlideCount?

Reply:
For example we have 10 product items. On the first glance we have 10 slides, but we know that on one slide we should have 6 items  slide. Result - 2 slides. So 10 items, but real 2 slides for them. So if you do not insist - real in this context is closer for me:)